app.json

```diff
@@ -77,10 +77,13 @@
     "ASHES_USERNAME": {
       "required": true
     },
-    "ROGUE_API_KEY": {
+    "AWS_S3_BUCKET": {
+      "required": true
+    },
+    "AWS_S3_URL": {
```

Comment:
Do review apps use S3? I'm curious if we should actually have these use the local filesystem `FILESYSTEM_DRIVER`, since we can't (automatically) provision a new S3 bucket for each review app & so they might clobber each other...

Reply:
Oh fair. I was only updating the variable name that was already there, but good to question whether it should even be there!

I checked against the [5.8 defaults](https://github.com/laravel/laravel/blob/5.8/config/filesystems.php#L16) and looks like `local` is set as the default but we have it set to `s3`. Probably something we changed a while back but I think defaulting to `local` makes more sense so I can remove the `AWS_*` env vars for review apps and if `FILESYSTEM_DRIVER` defaults to `local` we should be in a better place!

Reply:
Updated! I also realized there were a bunch of old `ASHES_*` env variables for the review apps which I [removed as well](https://github.com/DoSomething/rogue/pull/1071/commits/cddadb2cd8695add8ce6d8d7cc16b92074aee544).